docs/en/end-user/patch.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Patch Application
```

Comment:
Users never care about "Patch".

This should be sth like "Attach Sidecar" or other specific features, and these features should be built-in.